homeassistant/config_entries.py

```diff
@@ -1344,9 +1402,11 @@ async def async_initialize(self) -> None:
                 # New in 0.104
                 unique_id=entry.get("unique_id"),
                 # New in 2021.3
-                disabled_by=ConfigEntryDisabler(entry["disabled_by"])
-                if entry.get("disabled_by")
-                else None,
+                disabled_by=(
+                    ConfigEntryDisabler(entry["disabled_by"])
+                    if entry.get("disabled_by")
+                    else None
+                ),
```

Comment:
I think this is changed because you use black instead of ruff to format code in vscode. Please revert this change.